### PR TITLE
Color mode

### PR DESCRIPTION
**Pull request by AlexNechayev**
A dark mode toggle has been added

### CODE REVIEW

File: components/SiteFooter.vue [hunk 1]
```diff
@@ -44,18 +44,24 @@
...
-    <div class="py-6 text-center text-sm dark:text-gray-400">
-      <p class="text-xl font-bold">Official Vue.js Israel Community</p>
-      <span
-        >Built with 💚 by
-        <a href="https://www.linkedin.com/in/alex-nechayev/" target="_blank"
-          >Alex Nechayev</a
+    <div
+      class="grid grid-cols-1 grid-rows-1 gap-4 md:grid-cols-[100px_auto_100px]">
+      <div class="hidden items-center px-8 md:flex">
+        <ThemeToggler />
```
Comment (dimshik100) [inline components/SiteFooter.vue]: Please update to color mode or dark mode toggle component name.

File: components/SiteHeader.vue [hunk 2]
```diff
@@ -66,44 +66,47 @@ const closeMenu = () => {
...
-          to="/contact-us"
-          @click="closeMenu"
-          >Contact Us
-        </NuxtLink>
-      </li>
-    </ul>
-    <button class="flex justify-end p-4 md:hidden" @click="closeMenu">
-      <Icon name="mdi:close" size="60" />
-    </button>
+    class="sticky top-0 flex h-screen flex-col bg-white px-4 py-7 opacity-95 dark:bg-dark-bg-darker md:hidden">
```
Comment (dimshik100) [inline components/SiteHeader.vue]: Please fix menu to be on top of the header when open

Comment (AlexNechayev) [reply components/SiteHeader.vue]: Done

File: components/SiteFooter.vue (hunk 1 above)
Comment (AlexNechayev) [reply components/SiteFooter.vue]: Changed to ColorMode

**Review by dimshik100 — APPROVED** [review]

**Pull request merged**